pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "eavs-clc"
-version = "0.0.1"
+version = "0"
```

Comment:
Setting to `0` is just a common convention to indicate that the version is not meaningful.